### ISSUE TITLE
how does this relate to geospatial fabric

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/hydrofabric#4]
See link: https://www.sciencebase.gov/catalog/folder/5410bd38e4b07ab1cd98c45f

Pointed out by @dblodgett-usgs

also see http://cida-test.er.usgs.gov/nhdplus/geoserver/gwc/demo/nhdPlus:nhdflowline_network?gridSet=EPSG:900913&format=image/png

**Comment by dblodgett-usgs** [comment]
There is a lot in motion on this general topic right now. There will be a lot of new geospatial water-related services coming online in the next little while, starting with the stuff on the geoserver I put up and an ArcGIS server that the EPA put up. 

Roland's geospatial fabric for the National Hydrologic Modeling Framework could be a component. What's the best way to bring these things into scope for this work?

**Comment by jread-usgs** [comment]
I think it makes a lot of sense to open a collaboration that helps define the scope of this package. We have very basic needs: i.e., I want to get a watershed for a gaging station. I want to get a reach polyline for that gage too. I want to use the `whatNWISsites()` of `dataRetrieval` and do things w/ those sites, such as sample their geo properties or (at a very basic level) put the darn things on a map. 

How does this collaboration happen? It would be great on github, because that way other scientists and other agencies can be aware of it and contribute to it.

**Comment by ldecicco-USGS** [comment]
https://github.com/USGS-R/EGRET/issues/49

In the nearterm, it would be great for many to do what @jread-usgs just described.

**Comment by jread-usgs** [comment]
quick wins.

**Comment by dblodgett-usgs** [comment]
Agreed. I have NWIS gages indexed to NHDPlus, maybe that's a good place to start. 
http://cida-test.er.usgs.gov/nhdplus/geoserver/nhdPlus/wms?service=WMS&version=1.1.0&request=GetMap&layers=nhdPlus:gage&styles=&bbox=-124.956649780273,25.0479850769043,-66.9598922729492,49.1192016601562&width=795&height=330&srs=EPSG:4269&format=application/openlayers

http://cida-test.er.usgs.gov/nhdplus/geoserver/nhdPlus/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=nhdPlus:gage&maxFeatures=50&outputFormat=application/json

Lots of fun things we can do with these services. 
Lets get a list of 'quick win' stuff going.

full list of these services here: http://cida-test.er.usgs.gov/nhdplus/geoserver/web/?wicket:bookmarkablePage=:org.geoserver.web.demo.MapPreviewPage (don't hit the WMS for the big ones, it won't work.)

**Comment by jread-usgs** [comment]
Ok, this will help A LOT with linkages to NHD (`reachcode`) and NHD+ (`flcomid`)
